src/utils.py

```diff
@@ -275,3 +297,103 @@ def upload_experiment_to_s3(experiment_id: str,
     for upload_file in tqdm(upload_files):
         upload_file = str(upload_file)
         s3_client.upload_file(upload_file, bucket_name, upload_file.replace(current_global, object_global))
+
+
+def raven_to_csv_annotations(annotations_path: str,
+                             recording_path: str,
+                             dataset_name: str = 'recordings_2018_filtered'):
```

Comment:
Perhaps generate random name like wandb?

Reply:
The user can enter a name if they have a different dataset, but I think that since we publish it with our dataset as an example it can be simple to have the default as our dataset name. 
@mosheman5  what do you think?

Reply:
Either remove or keep the default, I see no use in using random name for dataset (the frequency of creating a dataset is much lower than the frequency of running an experiment)